core/src/main/java/io/questdb/cairo/sql/RowCursorFactory.java

```diff
@@ -54,6 +64,6 @@ default boolean isUsingIndex() {
         return false;
     }
 
-    default void prepareCursor(TableReader tableReader, SqlExecutionContext sqlExecutionContext) throws SqlException {
+    default void prepareCursor(TableReader tableReader) {
```

Comment:
when there is an intentional empty block like this I like to put at least a comment:

```
// no-op
```

so that I let other developers know it is intentional.

Reply:
Addressed in 1674eca52172845a7dcc050ae8d77e4ff0ce26a0